### DISCUSSION TITLE
Telegram Desktop rather than Telegram Web

### DISCUSSION THREAD

**Discussion by mv0hra** [ferdium/ferdium-app#1184] [Ideas]
Is there anyway or plans to have Telegram Desktop rather than Telegram Web. There are a lot of features that are missing in the Telegram Web. 
There are other softwares available, like Kotagram, but I like how you guys have created something that allows for the integration of other Social Theatre stuff.

**Comment by Alphrag** [comment]
Telegram Desktop is a fully fledge desktop app (as its name indicate), and not a webApp. So it does not really make sense to try to integrate it in Ferdium. We can only get access to what the WebApp offers, but you can choose to either use the `k` or `a` (formerly `z`) version of telegram.
